src/openrct2/Context.cpp

```diff
@@ -382,6 +382,82 @@ namespace OpenRCT2
             return LoadParkFromStream(&fs, path, loadTitleScreenOnFail);
         }
 
+        bool LoadParkFromStream(void * streamPtr, const std::string &path, bool loadTitleScreenFirstOnFail) final override
```

Comment:
You can change the parameter back to `IStream * stream`, it only needs to be upcasted in the C bridge function further down.